app/templates/components/footer-bar.hbs

```diff
@@ -1,8 +1,9 @@
 <div class="container">
-  {{#link-to "home"  (query-params author=null tag=null page=1) class="logo-font" }}
+  {{#link-to "home"  (query-params feed=null tag=null page=1) class="logo-font" }}
     conduit
   {{/link-to}}
   <span class="attribution">
-    An interactive learning project from <a href="https://thinkster.io">Thinkster</a>. Code &amp; design licensed under MIT.
+    An interactive learning project from <a href="https://thinkster.io">Thinkster</a>. Code &amp; design licensed under
```

Comment:
Was this changed by accident?

Reply:
This looks to be Prettier doing its thing, saving the file triggers this layout.